src/Utils/Apis/index.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
```

Comment:
Nice that you have util file, as an option `https://burger-api-xcwp.onrender.com/` can be initialized as constant